pkg/gardenlet/operation/shoot/types.go

```diff
@@ -87,6 +87,7 @@ type Shoot struct {
 	WantsAlertmanager                       bool
 	IgnoreAlerts                            bool
 	HibernationEnabled                      bool
+	VPNDisableNewImplementation             bool
```

Comment:
I would vote for a better name. Just "new VPN" (like the feature gate) or "new implementation" is not quite meaningful. What about something like `GolangBasedVPN`?

Reply:
Using same name `NewVPN` as for feature flag to reduce cognitive payload.